tests/usm/usm_get_pointer_queries.cpp

```diff
@@ -19,8 +19,12 @@ using namespace sycl_cts;
 
 template <sycl::usm::alloc alloc>
 void run_check(const sycl::queue &queue, sycl_cts::util::logger &log) {
-  const auto &device{queue.get_device()};
   const auto &context{queue.get_context()};
+  // According to the SYCL 2020 the first device in context should be
+  // used for alloc::host
```

Comment:
```suggestion
  // used for alloc::host.
```

Reply:
this comment has been applied in this commit: b8c581fe5eb0ed09444a50db00f9cc6c718a154a